worthapp/index.html

```diff
@@ -1291,7 +1291,7 @@ <h1 class="display-3">
   Beginning the Journey
  </h3>
  <div class="embed-responsive embed-responsive-16by9">
-  <video class="embed-responsive-item" controls="" poster="img/lucia.png">
+  <video class="embed-responsive-item" controls="" poster="img/.jpg">
```

Comment:
this video doesn't have a defined poster image yet - i guess i should leave it blank when that's the case but my templating system isn't that advanced yet